packages/core/lib/commands/debug/meta.js

```diff
@@ -59,7 +63,11 @@ module.exports = {
         option: "--compile-none",
         description:
           "Forces the debugger to use artifacts even if it detects a problem.  Dangerous; may cause errors."
-      }
+      },
+      {
```

Comment:
Maybe move this one up so that it displays earlier?  It's more useful than a dangerous option like `--compile-none`; it probably shouldn't be all the way at the end.  Maybe it could go after `--fetch-external`?

Reply:
Hm, this was marked resolved, but it appears to be in the same place still?